src/Components/Footer/SocialLink.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+interface SocialLinkT {
```

Comment:
prop타입에는 Prop을 포함하는게 일반적입니다
```suggestion
interface SocialLinkProps {
```